src/utils/RNPxlSearch.cpp

```diff
@@ -2311,6 +2311,7 @@ class RNPxlSearch :
     // reindex ids
     PeptideIndexing indexer;
     Param param_pi = indexer.getParameters();
+    param_pi.setValue("decoy_string", "rev_");

```

Comment:
We don't. However we have to supply ANY decoy string or else we get the "Unable to determine decoy string" error. There are no decoys in this test.
This specific case was always using the default decoy_string "DECOY_". When I fixed this I forgot that the default was "DECOY_", but for consistencies sake I should use "DECOY_".

However this and other cases are the origin of my question 1 here: https://github.com/OpenMS/OpenMS/pull/3454#issuecomment-390661536

If it s impossible to determine a decoy string (because there are no decoys) the function just errors out. In the old implementation it would simply take "DECOY_" and never use it. It didn't error out however.

Reply:
hmm @cbielow what do you thing. Shoud we, in the case the zero occurrences of any possible decoy in the was found just fall back to DECOY_ and just rely on the missing decoy warning that is produced later?